data_safe_haven/infrastructure/programs/declarative_sre.py

```diff
@@ -274,14 +274,14 @@ def __call__(self) -> None:
             "sre_remote_desktop",
             self.stack_name,
             SRERemoteDesktopProps(
-                aad_application_fqdn=networking.sre_fqdn,
-                aad_application_name=f"sre-{self.sre_name}-guacamole",
-                aad_auth_token=self.graph_api_token,
-                aad_tenant_id=self.cfg.shm.aad_tenant_id,
                 allow_copy=self.cfg.sre(self.sre_name).remote_desktop.allow_copy,
                 allow_paste=self.cfg.sre(self.sre_name).remote_desktop.allow_paste,
                 database_password=data.password_user_database_admin,
                 dns_server_ip=dns.ip_address,
+                entra_id_application_fqdn=networking.sre_fqdn,
```

Comment:
```suggestion
                entra_application_fqdn=networking.sre_fqdn,
```

Drop the `_id_` as an application is a feature

Reply:
Done